### PR TITLE
[CI](feat) Migrate the a5 test to the blue zone

### PR DESCRIPTION
**Pull request by xuedinge233**
## Summary

Enable A5 (Ascend 950) in the integration-test CI and retire the old
A5 wheel-build → BY-pipeline workflow line.

### CI workflows

- **Delete** `.github/workflows/Ascend950-wheels-build.yml` and
  `.github/workflows/Ascend950-pipeline-tests.yml` — A5 testing now runs in
  the standard Integration Tests NPU flow on the A5 runner; the separate
  build+trigger cascade is no longer used.
- **`runner-preparation.yml`**: the integration matrix now emits one entry
  per (chip × python), each carrying its full environment. A3 stays
  `cann 8.5.0 / py3.10+3.11 / linux-aarch64-a3-4`; A5 joins as
  `chip 950 / cann 9.1.0 / py3.12 / linux-amd64-a5-4` — chips are
  distinguished by runner.
- **`integration-tests-ascend.yml`**: the matrix is now fully config-driven
  (`cann_version` / `os` / `python_version` moved out of the global matrix
  into each config entry, since chips now differ).
- **`rebuild-tasks.yml`**: Remove the independent retry operations related to Ascend950-pipelining-tests
- **Per-arch compiler**: `cmake/bisheng-version.txt` now carries
  `aarch64` / `x86_64` entries (both ascendnpu-ir pr_2863 builds); the
  workflow selects the package by `uname -m` and keeps sha256 verification.
  The download adds retry + resume (`-C -`) — it was observed being killed
  mid-flight (exit 137) under runner resource limits.

### Image build

- **`docker/Dockerfile`**: torch-npu is now a build arg
  (`TORCH_NPU_VERSION`, default `2.7.1.post8` for the CANN 8.5.0 a3/910b
  line; A5 images pass `2.10.0.post4` per the CANN 9.1.0 compatibility table).
  The pin is removed from `requirements.txt`, which is shared by every chip.

### Tests

- **`test_elementwise_round.py`**: `standard_round` now computes in fp32
  like the kernel. The fp16/bf16-native reference rounded across `.5`
  boundaries (e.g. `ceil(-0.9997)` → `-1.0` in fp16), producing off-by-one
  mismatches. It only surfaced on CI because torch 2.10/2.12 generate
  different fp16 `randn` sequences for the same seed.

## Verification

- A5 `pytest_ut` with the pr_2863 x86 compiler package: **3525 passed /
  160 skipped**; the only remaining failures were the 3 round cases fixed
  here (probe confirms reference now agrees with the kernel at the failing
  index). Previous state: 327 failures (324 `bishengir-compile-a5` crashes
  + 3 round-boundary).
- A3 line unaffected: defaults keep cann 8.5.0 / py3.10+3.11 /
  torch 2.7.1.post8.

### CODE REVIEW

File: .github/workflows/integration-tests-ascend.yml [hunk 1]
```diff
@@ -9,21 +9,21 @@ on:
 
 jobs:
   integration-tests-ascend:
-    name: integration-tests-ascend (${{ matrix.config.name }}, py${{ matrix.python_version }})
+    name: integration-tests-ascend (${{ matrix.config.name }}, py${{ matrix.config.python_version }})
     runs-on: ${{ matrix.config.runs_on }}
     container:
```
Comment (LH-123L) [inline .github/workflows/integration-tests-ascend.yml]: https://github.com/triton-lang/triton-ascend/pull/1602  参考这个，吧TD也加到打包里面，后续会添加相关的case

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,3 @@
-torch-npu==2.7.1
```
Comment (hongziqi) [inline requirements.txt:1]: remain

File: third_party/ascend/unittest/pytest_ut/test_elementwise_round.py [hunk 3]
```diff
@@ -36,8 +36,12 @@ def standard_round_to_nearest_neighbor_even(x0):  # TO BE FIXED round to nearest
 
 
 def standard_round(x0):
-    res = torch.where(x0 >= 0, torch.floor(x0 + 0.5), torch.ceil(x0 - 0.5))
-    return res
+    # Compute in fp32 like the kernel: fp16/bf16 native arithmetic rounds
```
Comment (hongziqi) [inline third_party/ascend/unittest/pytest_ut/test_elementwise_round.py]: revert ut

File: requirements.txt (hunk 2 above)
Comment (xuedinge233) [reply requirements.txt:1]: Modified.

File: third_party/ascend/unittest/pytest_ut/test_elementwise_round.py (hunk 3 above)
Comment (xuedinge233) [reply third_party/ascend/unittest/pytest_ut/test_elementwise_round.py]: Modified.

**Review by 245516766 — APPROVED** [review]

**Review by hongziqi — APPROVED** [review]

**Review by WuTYSFG — APPROVED** [review]
